build/index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
-<html>
+<html lang='en-US' dir="ltr">
   <head>
     <meta charset="UTF-8" />
+    <meta http-equiv="content-type" content="text/html" charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>CDS Hooks Sandbox</title>
+    <link rel='stylesheet' type='text/css' href='./styles.css'>
   </head>
   <body>
-  <div id="root"></div>
-  <script src='./fhir-client.min.js'></script>
-  <script src="./app.bundle.js"></script>
+    <div id="root"></div>
+    <script src='./fhir-client.min.js'></script>
```

Comment:
We should mention somewhere in say the ReadMe or add an additional file indicating which version/tag of the fhir-client we are pulling unless there exists a CDN for pulling this in. 
We don't have to update it now but a suggestion for future.